src/App.tsx

```diff
@@ -1,61 +1,124 @@
 import './App.scss';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList, Todo } from './components/TodoList/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const firstTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(u => u.id === todo.userId),
+}));
 
 export const App = () => {
+  const [submitTitleError, setSubmitTitleError] = useState(false);
+  const [submitUserError, setSubmitUserError] = useState(false);
+  const [todos, setTodos] = useState<Todo[]>(firstTodos);
+  const [titleInput, setTitleInput] = useState('');
+  const [selectedUser, setSelectedUser] = useState('0');
```

Comment:
```suggestion
  const [selectedUserId, setSelectedUserId] = useState('');
```
you save only Id here
Also, you can use `''` empty string as an empty value to make all checks simpler (e.g. `!selectedUserId` instead of `selectedUserId !== '0'`)